tests/integration/components/ember-popper/action-test.js

```diff
@@ -11,42 +11,46 @@ if (hasEmberVersion(1, 13)) {
 
   test('it calls onCreate', function(assert) {
     assert.expect(2);
+
     let called = 0;
-    const action = (data) => {
+    this.on('create', (data) => {
       called++;
       assert.ok(data && data.instance, 'onCreate action is called with dataObject');
-    };
-    this.on('create', action);
+    });
+
     this.render(hbs`
-    {{#ember-popper onCreate=(action "create")}}
-      template block text
-    {{/ember-popper}}
-  `);
+      {{#ember-popper onCreate=(action "create")}}
+        template block text
+      {{/ember-popper}}
+    `);
 
     return wait()
       .then(() => assert.equal(called, 1, 'onCreate action has been called'));
   });
 
   test('it calls onUpdate', function(assert) {
     assert.expect(2);
+
     let called = 0;
-    const action = (data) => {
+    this.on('update', (data) => {
       called++;
       assert.ok(data && data.instance, 'onUpdate action is called with dataObject');
-    };
-    this.on('update', action);
+    });
+
     this.render(hbs`
-    {{#ember-popper onUpdate=(action "update")}}
-      template block text
-    {{/ember-popper}}
-  `);
+      {{#ember-popper onUpdate=(action "update")}}
+        template block text
+      {{/ember-popper}}
+    `);
 
     return wait()
       .then(() => triggerEvent(document.querySelector('body'), 'scroll'))
+      // triple-wait because RAF is hard to wait for consistently
```

Comment:
It looks like we haven't registered the RAF-scheduler test waiter, that could be causing more noise

Reply:
Ah, didn't realize we haven't added the RAF waiter here. I'll add it in and experiment with the `wait()`s.